src/main/java/emissary/test/core/ExtractionTest.java

```diff
@@ -330,6 +358,136 @@ protected void checkStringValue(Element meta, String data, String tname) {
         }
     }
 
+    /**
+     * Method to validate a meta key name against fields/params within Filter .cfg files. Currently, this is set to NOT RUN
+     * unless the meta field specifies it in the .xml file like: meta validateField="true"
+     *
+     * Along with this, filter paths must be passed through the .xml within the answers section. This is demonstrated within
+     * the TestValidateFieldExtractionTest.xml file within resources/emissary/test/core
+     *
+     * @param key the current meta key name to validate against LogFilter
+     * @param tname the current test name
+     */
+    protected void validateFieldInFilter(String key, String tname) {
```

Comment:
Sonar recommends breaking this method up to reduce complexity. Parts can possible be abstracted out into helper methods.
Not a blocker but something to be aware of.